### ISSUE TITLE
Make sure blink retries on ETIMEDOUT errors to gambit

### ISSUE THREAD

**Issue by sergii-tkachenko** [DoSomething/blink#55]
May 10th 2017, 05:20:18.741	b6689b63-a5c6-4ac3-9ba7-bd1a8d5d7481	at=warning application=blink code=message_processing_error queue=gambit-chatbot-mdata request_id=b6689b63-a5c6-4ac3-9ba7-bd1a8d5d7481 FetchError: request to http://ds-mdata-responder.herokuapp.com/v1/chatbot failed, reason: connect ETIMEDOUT 23.23.77.117:80

**Comment by sergii-tkachenko** [comment]
@rapala61 This is an edge case. Still, better to check on.

**Comment by rapala61** [comment]
I think I saw this happen during Rogue outage 1/10/18.

**Comment by sergii-tkachenko** [comment]
Yes, this happens pretty rarely, but it does happen. I spotted it a couple of times since last May.
